ups/validate_base.table

```diff
@@ -0,0 +1,8 @@
+setupRequired(pyyaml)
+setupRequired(numpy)
+setupRequired(astropy)
+
+setupOptional(base)
```

Comment:
Ah ok. So when I wrap `import lsstimport` in a try/except, `validate_base` isn't even optionally depending on `base`.